### PR TITLE
feat: Run csharp/il2cpp events through symbolicator

### PR DESCRIPTION
**Pull request by Swatinem**
This will send csharp/il2cpp events through the native event processor, so events are being symbolicated.

### CODE REVIEW

File: src/sentry/lang/native/utils.py [hunk 1]
```diff
@@ -12,7 +12,9 @@
 WINDOWS_PATH_RE = re.compile(r"^([a-z]:\\|\\\\)", re.IGNORECASE)
 
 # Event platforms that could contain native stacktraces
-NATIVE_PLATFORMS = ("cocoa", "native")
+# il2cpp events have the "csharp" platform, and we do need to run those
+# through symbolicator to correctly symbolicate il2cpp stack frames.
+NATIVE_PLATFORMS = ("cocoa", "native", "csharp")
```
Comment (priscilawebdev) [inline src/sentry/lang/native/utils.py:17]: just to confirm...we shouldn't update this in the [UI](https://github.com/getsentry/sentry/blob/ba3ecb9c6aba5b8f027817208f1327816d62b67f/static/app/utils/displayReprocessEventAction.tsx#L11) too right?

File: static/app/utils/displayReprocessEventAction.tsx [hunk 2]
```diff
@@ -56,7 +56,7 @@ function getStackTracePlatforms(event: Event, exceptionEntry: EntryException) {
   ]);
 }
 
-// Checks whether an event indicates that it has an apple crash report.
+// Checks whether an event indicates that it is a native event.
```
Comment (priscilawebdev) [inline static/app/utils/displayReprocessEventAction.tsx:59]: thank you!

File: src/sentry/lang/native/utils.py (hunk 1 above)
Comment (Swatinem) [reply src/sentry/lang/native/utils.py:17]: I don’t think so.

**Review by priscilawebdev — APPROVED** [review]
Should we add tests for these changes too?

code looks good to me

**Comment by Swatinem** [comment]
I added a unit test, and noticed we should prefer the existing C# `package` instead of overriding with the one coming from symbolicator.

**Review by loewenheim — APPROVED** [review]

**Pull request merged**